app/javascript/packs/commission_histories/index_template.vue

```diff
@@ -45,27 +45,26 @@
 
 <script>
   import axios from 'axios'
+  import { mapGetters } from 'vuex';
 
   axios.defaults.headers.get["Authorization"] = window.api_authorization
 
   export default {
-    props: ['query', 'network'],
+    props: ['query'],
     data () {
-      if(this.query && !this.query == ''){
-        var api_url = '/api/v1/commission-changes/' + this.network + '?query=' + this.query + '&'
-      } else {
-        var api_url = '/api/v1/commission-changes/' + this.network + '?'
-      }
       return {
         commission_histories: [],
         page: 1,
         total_count: 0,
         sort_by: 'created_at_desc',
-        api_url: api_url,
+        api_url: null,
         account_name: this.query
       }
     },
     created () {
+      this.api_url = this.query && !this.query == ''
```

Comment:
This is waaay to long condition for one-liner syntax (? :). I know it's not exactly in the scope of this story, but you could also fix that.